skimage/feature/util.py

```diff
@@ -41,7 +41,8 @@ def extract(self, image, keypoints):
 
 
 def plot_matches(ax, image1, image2, keypoints1, keypoints2, matches,
```

Comment:
I feel that this is not the right approach. Do we have any alternatives?
I think this should be done by passing several axes via `ax` parameter or something, but, in that case, it won't be possible to draw the lines.

@ahojnnes would you like to comment?

Reply:
Since multiple axes are not possible, I think a kwarg is correct, but `left_right` doesn't work for me... I would suggest `alignment={'horizontal','vertical'}`